proposed/2023/NuGetAudit-without-nuget.org-as-packageSource.md

```diff
@@ -0,0 +1,221 @@
+# NuGetAudit without nuget.org as a package source
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- Author: [zivkan](https://github.com/zivkan)
+- GitHub Issue: [12698](https://github.com/NuGet/Home/issues/12698)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+This proposal adds a new `<auditSource>` to *NuGet.Config* files, allowing *NuGetAudit* to obtain known vulnerabilities without adding nuget.org as a package source.
```

Comment:
How will this impact the browse/updates tab in PM UI (vulnerable adornments) and the `dotnet list --vulnerable` command?

Reply:
`dotnet list --vulnerable` uses package metadata (registration), not the vulnerability info resource. Therefore I consider it out of scope for this feature spec.

PM UI uses the vulnerability info resource, but only for installed, transitive packages. That code should be updated, so thanks for the reminder.

Reply:
It might be worth mentioning the `dotnet list` limitation. If effectively means that `dotnet list --vulnerable` is stuck on `<packageSource>` because of a protocol difference. It might not be apparent to users that a VDB can't support `dotnet list`.